.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
+labels: bug
+---
+
+<!-- Lines like this are comments and will be invisible -->
+
+# Bug / performance issue / build issue
```

Comment:
At what point performance matters go from being bugs to being feature requests?

Reply:
Don't know - we've also not had many in the past. Something we can play by ear for now I think